lib/package-manager.coffee

```diff
@@ -432,6 +432,13 @@ class PackageManager
       repoUrl = "https://github.com/#{repoName}"
     repoUrl.replace(/\.git$/, '').replace(/\/+$/, '').replace(/^git\+/, '')
 
+  getRepositoryBugUri: ({metadata}) ->
+    {bugs} = metadata
+    bugUri = bugs ?.url ? bugs ?.email ? this.getRepositoryUrl({metadata}) + '/issues/new'
```

Comment:
I do not really like that line 🤔

Reply:
There seems to be a problem with your implementation as the [NPM docs](https://docs.npmjs.com/files/package.json#bugs) says that `bugs` may be specified as an object containing a URL and/or email, or if it's only an URL it may be specified as a string directly. What exactly does metaData containg?

Reply:
I took the liberty of modifying your solution to adhere to the NPM package specification, see #1093.